plugins/modules/activation_key.py

```diff
@@ -62,12 +62,15 @@
         description:
           - Pool ID of the Subscription to be added.
           - Mutually exclusive with I(name) and I(upstream_pool_id).
+          - Also named I(Candlepin Id) in the CSV export of the subscriptions,
+          - it is as well the UUID as output by I(hammer subscription list).
```

Comment:
```suggestion
          - Also named C(Candlepin Id) in the CSV export of the subscriptions,
          - it is as well the C(UUID) as output by C(hammer subscription list).
```

Is the "CSV" export still a thing in modern Satellite/Katellos?

Reply:
And, yes, in Satellite 6.8 / Katello 3.16, CSV export is still a thing, even if I'm not sure it's modern :-)

![image](https://user-images.githubusercontent.com/3105685/104457326-306a3480-55aa-11eb-8410-8238b52b1750.png)

Reply:
Oh, cool. I thought about `hammer csv` which is gone. :+1: